src/internal.c

```diff
@@ -1555,10 +1555,12 @@ static int ImportOptions(WOLFSSL* ssl, const byte* exp, word32 len, byte ver,
         idx++;
 #endif
     }
-
+#ifdef WOLFSSL_DTLS
```

Comment:
Let's make sure we have build coverage on this. Please add 

Please add `--enable-sessionexport` and `--enable-sessionexport --enable-dtls --enable-dtls13` to `.github/workflows/os-check.yml`